doc/source/operations/openstack-reconfiguration.rst

```diff
@@ -0,0 +1,148 @@
+
+   kayobe# ansible-vault edit ${KAYOBE_CONFIG_PATH}/secrets.yml --vault-password-file=<Vault password file path>
+
+Concatenate the contents of the certificate and key files to create
+``secrets_kolla_external_tls_cert``.  The certificates should be installed in
+this order:
+
+* TLS certificate for the public endpoint FQDN
+* Any intermediate certificates
+* The TLS certificate private key
+
+In ``${KAYOBE_CONFIG_PATH}/kolla.yml``, set the following:
+
+.. code-block:: yaml
+
+   kolla_enable_tls_external: True
+   kolla_external_tls_cert: "{{ secrets_kolla_external_tls_cert }}"
+
+To apply TLS configuration, we need to reconfigure all services, as endpoint URLs need to
+be updated in Keystone:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud service reconfigure
+
+Alternative Configuration
+-------------------------
+
+As an alternative to writing the certificates as a variable to
+``secrets.yml``, it is also possible to write the same data to a file,
+``$KAYOBE_CONFIG_PATH/kolla/certificates/haproxy.pem``.  The file should be
+vault-encrypted in the same manner as secrets.yml.  In this instance,
+variable ``kolla_external_tls_cert`` does not need to be defined.
+
+See `Kolla-Ansible TLS guide
+<https://docs.openstack.org/kolla-ansible/latest/admin/tls.html>`__ for
+further details.
+
+Updating External TLS Certificates
+----------------------------------
+
+Check the expiry date on an installed TLS certificate from a host that can
+reach the OpenStack APIs:
+
+.. code-block:: console
+
+   openstack# openssl s_client -connect <Public endpoint FQDN>:443 2> /dev/null | openssl x509 -noout -dates
+
+.. note::
+
+   Prometheus Blackbox monitoring can check certificates automatically
+   and alert when expiry is approaching.
+
+To update an existing certificate, for example when it has reached expiration,
+change the value of ``secrets_kolla_external_tls_cert``, in the same order as
+above.  Run the following command:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud service reconfigure --kolla-tags haproxy
+
+.. _taking-a-hypervisor-out-of-service:
+
+Taking a Hypervisor out of Service
+==================================
+
+To take a hypervisor out of Nova scheduling:
+
+.. code-block:: console
+
+   admin# openstack compute service set --disable \
+          <Hypervisor name> nova-compute
+
+Running instances on the hypervisor will not be affected, but new instances
+will not be deployed on it.
+
+A reason for disabling a hypervisor can be documented with the
+``--disable-reason`` flag:
+
+.. code-block:: console
+
+   admin# openstack compute service set --disable \
+          --disable-reason "Broken drive" <Hypervisor name> nova-compute
+
+Details about all hypervisors and the reasons they are disabled can be
+displayed with:
+
+.. code-block:: console
+
+   admin# openstack compute service list --long
+
+And then to enable a hypervisor again:
+
+.. code-block:: console
+
+   admin# openstack compute service set --enable \
```

Comment:
Generally speaking, we should get rid of these admin# and kayobe# prefixes. I think they were used in the admin guide but might render weirdly in the wiki
```suggestion
   openstack compute service set --enable \
```